pkg/secret/store.go

```
@@ -105,6 +106,28 @@ func (s *store) Create(params CreateParams) error {
 	return s.appendAndSave(sf, params)
 }
 
+// List reads secrets.json and returns metadata for all stored secrets.
+func (s *store) List() ([]ListItem, error) {
+	sf, err := s.loadSecretsFile()
+	if err != nil {
+		return nil, err
+	}
+	items := make([]ListItem, 0, len(sf.Secrets))
+	for _, rec := range sf.Secrets {
+		items = append(items, ListItem{
```

golangci-lint (2.11.4)

[error] S1016: should convert rec (type secretRecord) to ListItem instead of using struct literal

(staticcheck)